README.Rmd

```diff
@@ -6,7 +6,7 @@
 
 ```{r setup, echo=FALSE}
 # set global chunk options: images will be 7x7 inches
-knitr::opts_chunk$set(fig.width=7, fig.height=7, fig.path="figs/", warning=FALSE)
+knitr::opts_chunk$set(fig.width=7, fig.height=7, fig.path="man/figs/", warning=FALSE)
```

Comment:
I think the `man/` here is bad. This will put figures in the manpage folders, which we definitely don't want to do. Probably best to Rbuilignore `figs/`